ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/initial/.content.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is not part of the scope of the ticket

Reply:
removed